index.html

```diff
@@ -159,141 +156,401 @@ <h2>
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>pointer lock state</dfn> definition
+        </h3>
+        <p>
+          The [=pointer lock state=] is the state where a single DOM element,
+          which we will call the <dfn>pointer-lock target</dfn>, receives all
+          mouse events and the cursor is hidden.
+        </p>
+        <p>
+          Once in the [=pointer lock state=] the [=user agent=] has a
+          [=pointer-lock target=], a <dfn>pointer-lock options</dfn>, which is
+          a {{PointerLockOptions}} and a <dfn>cursor position</dfn> which is a
+          pair of numbers representing the location of the system mouse cursor
+          when the Pointer Lock State was entered (the same location that is
+          reported in `screenX`, `screenY`). The [=pointer-lock target=] must
+          receive all relevant user generated {{MouseEvent}} events: namely,
+          all user-generated `mousemove`, `mousedown`, `mouseup`, `click`,
+          `dblclick`, `auxclick`, and `wheel` [[ui-events]]. No other elements
+          may receive these events while in [=pointer lock state=]. There will
+          be no dispatching of events that require the concept of a mouse
+          cursor: namely, `mouseenter`, `mouseleave`, `mouseover`, `mouseout`,
+          `drag`, and `drop`.
+        </p>
+        <p>
+          While in the [=pointer lock state=] if the [=pointer-lock options=]'
+          {{PointerLockOptions/unadjustedMovement}} member is `true` the event
+          coordinates must not be affected by the underlying platform behaviors
+          such as mouse acceleration. In other words, UA must use the APIs
+          provided by the underlying platform to guarantee getting the raw
+          events. If the {{PointerLockOptions}}'
+          {{PointerLockOptions/unadjustedMovement}} member is `false` UA must
+          rely on the default behavior of the underlying platform regarding the
+          mouse acceleration.
+        </p>
+        <p>
+          In the [=pointer lock state=] the system mouse cursor must be hidden.
```

Comment:
We might need some clarity on some aspects here that it's left to the underlying OS.

Reply:
How about:
```
In the [=pointer lock state=], the system mouse cursor is hidden and the window is prevented from losing focus, regardless of mouse movement or button presses. This is directly or indirectly achieved utilizing the underlying operating system API.
```

Reply:
The clarification sounds good.

Reply:
Done